### PR TITLE
✨ Initial family/family_relationship conformance resources/mappers

### PR DESCRIPTION
**Pull request by liberaliscomputing**
This PR closes:
  - d3b-center/clinical-data-flow#75
  - d3b-center/clinical-data-flow#78

### CODE REVIEW

File: site_root/input/resources/examples/Patient-pt-002.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          "valueReference": {
+            "reference": "Patient/pt-001"
+          }
+        },
+        {
+          "url": "relation",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
```
Comment (gsantia) [inline site_root/input/resources/examples/Patient-pt-002.json:71]: this is a general comment; when I see `CHILD`, `child`, and `Child` all right next to each other, it kind of bugs me

Comment (liberaliscomputing) [reply site_root/input/resources/examples/Patient-pt-002.json:71]: @gsantia, good catch! So, if you look into this [ValueSet](https://www.hl7.org/fhir/v3/FamilyMember/vs.html), `code` and `display` are pre-defined. And, if you look into the general description of [CodeableConcept](https://www.hl7.org/fhir/datatypes.html#CodeableConcept), `code` and `display` come from `system` while `text` is a plain, source text description. Say, we have a `"Type 2 diabetes"` diagnosis for a patient. This concept is widely used, but may be expressed in different forms from different ontologies. In SNOMED CT, this is described as `"Diabetes mellitus type 2"` with code `"44054006"`. In HPO, it is described as `"Type II diabetes mellitus"` with code `"HP:0005978"`. In this case, we may want to create the following attribute, using CodeableConcept:

```json
"Condition.code": {
  "coding": [
    {
      "system": "URL of SNOMED CT",
      "code": "44054006",
      "display": "Diabetes mellitus type 2"
    },
    {
      "system": "URL of HPW",
      "code": "HP:0005978",
      "display": "Type II diabetes mellitus"
    },
  ],
  "text": "Type 2 diabetes"
}
```

This way, even though how idiosyncratic a form of a diagnosis is described across multiple systems and by asserter, we can always harmonize multiple expressions referring to the sample concept.

File: kf_model_fhir/mappers/resources/group.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            'resourceType': RESOURCE_TYPE,
+            'id': make_identifier(RESOURCE_TYPE, study_id, family_id),
+            'meta': {
+                'profile': [
+                    'http://hl7.org/fhir/StructureDefinition/Group'
+                ]
+            },
+            'identifier': [
+                {
+                    'system': f'https://kf-api-dataservice.kidsfirstdrc.org/families?study_id={study_id}',
```
Comment (fiendish) [inline kf_model_fhir/mappers/resources/group.py:43]: Should this be `'system': f'https://kf-api-dataservice.kidsfirstdrc.org/families?external_id=',` ?

Comment (liberaliscomputing) [reply kf_model_fhir/mappers/resources/group.py:43]: For other mappers like [this] (https://github.com/kids-first/kf-model-fhir/blob/master/kf_model_fhir/mappers/resources/kfdrc_patient.py#L142-L143), I made the system contextualized within a study under a certain endpoint. But, in the [new plugin](https://github.com/kids-first/kf-model-fhir/blob/ingest-plugin/kf_model_fhir/ingest_plugin/kfdrc_patient_no_phi.py#L139-L140), I figured you made it point to that specific entity. For the current mappers, I may use the previous approach for the consistency across all mappers and take the new approach for the plugin development?

Comment (fiendish) [reply kf_model_fhir/mappers/resources/group.py:43]: You're right, it should also include the study

Comment (liberaliscomputing) [reply kf_model_fhir/mappers/resources/group.py:43]: Updated!

File: site_root/input/resources/extensions/StructureDefinition-relation.json [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        "min": 1,
+        "max": "1",
+        "type": [
+          {
+            "code": "CodeableConcept"
+          }
+        ],
+        "binding": {
+          "strength": "required",
+          "description": "A relationship between two people characterizing their 'familial' relationship.",
```
Comment (znatty22) [inline site_root/input/resources/extensions/StructureDefinition-relation.json:113]:
```suggestion
          "description": "A relationship between two people characterizing their 'biological' relationship.",
```

Comment (liberaliscomputing) [reply site_root/input/resources/extensions/StructureDefinition-relation.json:113]: Updated!

**Pull request merged**